source/references/UI_spec.rst

```diff
@@ -217,7 +217,11 @@ similar to the selection of other WS data objects.
 
 :service_params: The parameters supplied to the dynamic service call as JSON. The special text
                  ``"{{dynamic_dropdown_input}}"`` will be replaced by the value from the user input
-                 at call time.
+                 at call time. Additionally, any keywords defined as parameters for the app will be
+                 replaced with the user input value at call time. For instance, if an app has a text
+                 parameter called ``"output_name"`` and and the user has specified the output name
```

Comment:
Maybe change "called" to "with id" or something similar to make it extra clear we're talking about app parameters by id in spec.json.